src/components/workspace/Workspace.js

```diff
@@ -381,6 +387,49 @@ export default class Workspace extends Base {
     return createdNode;
   }
 
+  addConditionLabel(direction, isTemp) {
+    if (!this.component) {
+      return this.ce('span');
+    }
+    let labelText = ''
+    if (this.component.out.length === 0) {
+      labelText = 'Yes';
+    } else if (this.component.out.length === 1) {
+      labelText = 'No';
+    }
```

Comment:
use ternary instead of checking length twice.
`labelText = this.component.out.length === 0 ? 'yes': 'no'`;